polkadot-wiki/sidebars.js

```diff
@@ -43,14 +43,16 @@ module.exports = {
           items: [
             {
               type: "category",
-              label: "Accounts",
+              label: "Accounts and Identity",
               items: [
                 "learn/learn-accounts",
                 "learn/learn-account-generation",
                 "learn/learn-balance-transfers",
                 "learn/learn-extrinsics",
                 "learn/learn-account-multisig",
                 'learn/learn-proxies',
+                "learn/learn-identity",
```

Comment:
Isn't it confusing having identity here? I see this as not a basic information, and having accounts and identity together can be confusing. What do you think?